users.go

```diff
@@ -89,6 +89,9 @@ type User struct {
 	PrivateProfile            bool               `json:"private_profile"`
 	SharedRunnersMinutesLimit int                `json:"shared_runners_minutes_limit"`
 	CustomAttributes          []*CustomAttribute `json:"custom_attributes"`
+
+	// UsingLicenseSeat is only available in GitLab Starter, Bronze, or higher.
+	UsingLicenseSeat *bool `json:"using_license_seat,omitempty"`
```

Comment:
Maybe also add `extra_shared_runners_minutes_limit` while you are at it?

But also, please do not make a distinction for certain fields by making them a pointer field. See my comment here #998 as well. So please make it normal fields and place them below `SharedRunnersMinutesLimit` following the order in the docs.

Thanks!

Reply:
I see your point, to be honest am little bit confused: some codes use pointer field. Sure, I can update this.

Do we want to have `omitempty` at `UsingLicenseSeat` field?

```go
	SharedRunnersMinutesLimit      int                `json:"shared_runners_minutes_limit"`
	ExtraSharedRunnersMinutesLimit int                `json:"extra_shared_runners_minutes_limit"`
	UsingLicenseSeat               bool               `json:"using_license_seat,omitempty"`
```

Reply:
Nope, no `omitempty` for those please.

As for what should be a pointer and what should not. In options structs everything should be a pointer (so we can use them properly when making API calls) and in response structs only struct fields are pointers. Hope that helps...

Reply:
Gotcha, thank you for your explanation. Updated the code as you rested - force pushed the branch.